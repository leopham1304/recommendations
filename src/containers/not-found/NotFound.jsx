import React, { memo } from "react";
import "./NotFound.scss";

const NotFound = () => {
  return <h1 className="notfound-container">Page Not Found</h1>;
};

export default memo(NotFound);
