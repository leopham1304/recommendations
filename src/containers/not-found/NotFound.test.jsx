import React from "react";
import NotFound from "./NotFound";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("NotFound", () => {
  const wrapper = shallow(<NotFound />);

  it("should render the NotFound correctly", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
