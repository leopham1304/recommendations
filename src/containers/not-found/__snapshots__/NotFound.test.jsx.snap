// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render the NotFound correctly 1`] = `
<h1
  className="notfound-container"
>
  Page Not Found
</h1>
`;
